0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+
+```{r identify-feature, fig.cap = 'How to see feature info in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/identify_feature.png")
+```
+ 
+  3. Copy information from a feature. Select the row then hover your cursor over the text you want to copy then `cmd+c` to copy.
+
+```{r copy-feature, fig.cap = 'How to copy information from a features in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/copy_feature_info.png")
+```
+
+  4. Creating maps with `Projects/print layout`. This is where you can create a map that can be exported as an image or pdf. Below is an example of a map layout. Check out [this video](https://www.youtube.com/watch?v=OlDGEVhpmo0) for how to make maps.
+
+```{r qgis-maps, fig.cap = 'How to create and export maps in QGIS',eval=T}
+knitr::include_graphics("fig/mapping/qgis_maps.png")
+```
+
+
+### Editing a Project
+
+- Sync the project [following these instructions](https://merginmaps.com/docs/manage/synchronisation/#how-to-synchronise-changes-in-mergin-maps) before making any edits to make sure you have the most up to date version of the project. Others may have edited the project since you last opened it.
+
+- Make sure to sync the project [following these instructions](https://merginmaps.com/docs/manage/synchronisation/#how-to-synchronise-changes-in-mergin-maps) after making any edits.
+
+
+### Using Mergin on a phone
+
+- We find that Mergin works much better (faster rendering) with *airplane* mode on.    
+
+### Field Data Entry into Mergin
+
+- Over the last year we have seen uncommon issues with syncing and data corruption that provide even more incentive to be sure to do paper copies of the forms as well. 
+- There are links to the methods docs in the forms with the intention that you can save them to your phone.  Obviously, the provincial method documents are key.
+- Switching between themes is very useful in the field. You can add more in QGIS.
+- Site ID is important:
+  - If a site has a PSCIS ID then that is used as the identifier. 
+  - If there is no PSCIS ID we use the "Crossing - Modelled ID" (a.k.a "My Crossing Reference") as the "Modelled ID". If there is neither (rare but usually occurs when a stream is not mapped) we use a numeric ID based on the date (YYYYMMDD01 - example 2023070502 - would be our second assessment for today).
+  - If there is an associated chris_culvert_id ("MoTi" identifier) we include that as well. We take a photo of the road with the PSCIS ID or Modelled ID in the corner and upload photos to mergin that are
+- When we first arrive at a site we take a photo of the road with the PSCIS ID or Modelled ID in the corner. This allows easy reference to which photos are for which sites. When we are done filling out the paper form we take a photo of it (for data backup and to make it easy to see which photos are from where).
```

Comment:
where do these photos go? (take on camera with timestamp (where do you get that? - app store) in landscape with proper dimensions (look at timestamp settings).  Screenshots of how to setup timestamp properly (date format, utm with proper crs (pretty sure the crs type we use is documented). Photo dimensions matter.  Look at setting options (can't remember - might be written somewhere) and note what aligns with my million photos over the years.

saw somewhere you weren't sure where in zotero the word doc was with the timestamp details.  Best to note where you looked since writing that down might trigger potential ideas about where it is and help get familiar with all the nooks and crannies. Experience and exploration usually trumps direction and saves time for me. its in Zotero/Fisheries/Fish Passage/methods